transports.md

```diff
@@ -1,8 +1,14 @@
 # Transports
-
-* Portions that are transport agnostic
-* Application to transports
-* Http
-* Websockets
-* Quic
-* Transport binding
+## Summary
+This spec details how different transports can be used to transmit DIDComm messages.
+## Motivation
+The DIDComm message is constructed as JWM, using JWE for confidentiality and authentication, so it is transport agnostic.
+Each transport has unique features and properties, so we need to define a standard usage that will be aligned with the asynchronous+simplex design decision.
+## Reference
+### HTTP(S)
+The POST method is used to send the DIDComm message.
+The POST message MIME type is application/didcomm-enc-env, see RFC 0044: DIDComm File and MIME Types for more details.
```

Comment:
I'd prefer that we not reference Aries RFCs docs for back history. Rather, I'd prefer that we take out the relevant details to conform with the spec and reference to the spec instead.

Reply:
Done